### ISSUE TITLE
Getting Nan after certain epochs

### ISSUE THREAD

**Issue by ajaysinj117** [guigzzz/Keras-Yolo-v2#4]
I started the training however after certain epochs i am getting loss NaN . just before getting nan the loss was converging .
